### ISSUE TITLE
Abnormal frame is on honor magic4 120Hz，like skipping some frame

### ISSUE THREAD

**Issue by best2142** [flyinghead/flycast#572]
**Please Note: This form is the minimum required information for submitting bugs.**  
**Removing this form may lead to your issue being closed until it is completed.**

Platform / OS / Hardware: 
Android 12 MAGIC UI 6.0
Github hash: 
ltpo? but dynamic refresh rate is fine,it happing on 90HZ and 120HZ
Hardware: 
honor magic4
**_Description of the Issue_**
it like skipping some frame，Graphic discontinuity，and duplicate frames is unable
Add a short, concise description of the issue here

**_Debugging Steps Tested_**

1.set 120HZ on refresh rate
2.play any game（Daytona USA is obvious）

**_Logs Gathered_**
sorry ，i don‘t know how to get logs

**_Screenshots_**

(Replace this line with any issue screenshots)

**Comment by flyinghead** [comment]
Yes, I have the same issue with an Asus ZenFone 8 (Adreno 660). Adaptive refresh rate works fine (at 60 Hz) but forcing 120 Hz produces duplicate frames artifacts, even when using a swap interval of 2.

For now, stay on 60 Hz for Flycast.
